### PR TITLE
[Excel] (Custom Functions) Minor changes + whitespace adjustments to custom-functions-json.md

### PR DESCRIPTION
**Pull request by alison-mk**
This PR makes the following changes: 
- Clarifies the page title + incorporates purpose of page into title 
- Makes 'autogeneration' term consistent with internal guidance and other docs pages (by removing dash in auto-generation)
- Clarifies which code sample line 240 refers to 
- Cleans up some whitespace issues, adds a period, rephrases a confusing sentence

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[e2584ca](https://github.com/OfficeDev/office-js-docs-pr/commits/e2584cab238f6a230ef5967858878f64015437b4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-custom-functions-json-cleanup/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2273) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010192332445733-2273%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=p2bGI0OCoafnQg7mdunoz4Y68zzj7jbsUcXgKcMtaOQ%3D&st=2020-10-19T23%3A28%3A54Z&se=2020-11-19T23%3A33%3A54Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f0badda](https://github.com/OfficeDev/office-js-docs-pr/commits/f0badda8011bb582ac296d4f97756e19d4c977be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-custom-functions-json-cleanup/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2273) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010200039291254-2273%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=g9odkPL30y%2FLZtf5e5tlB%2B9lcoMO2tpnP1JH5DoF6Mc%3D&st=2020-10-20T00%3A35%3A07Z&se=2020-11-20T00%3A40%3A07Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2c787a5](https://github.com/OfficeDev/office-js-docs-pr/commits/2c787a52af11c4740906ea0aadb94048be172c99)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-custom-functions-json-cleanup/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2273) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010200043005003-2273%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Mvbcd8zgDxQbyd%2FCdUcCcz1kGLTqjAiSxV7GKveBUCo%3D&st=2020-10-20T00%3A38%3A42Z&se=2020-11-20T00%3A43%3A42Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c6e4cec](https://github.com/OfficeDev/office-js-docs-pr/commits/c6e4cec5df77161c9c7a35028c8089026740da5a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-custom-functions-json-cleanup/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2273) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010200054573389-2273%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=fsdOudZ0u7C1GK8OWuK2JYtMHNv%2FpEGC4B2hW%2FymkMk%3D&st=2020-10-20T00%3A50%3A33Z&se=2020-11-20T00%3A55%3A33Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ef153b8](https://github.com/OfficeDev/office-js-docs-pr/commits/ef153b88d7bec99e523aef4d8997fa4d4353aae8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-custom-functions-json-cleanup/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2273) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010222014371823-2273%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Q3spJTWxBjNIsymqPgMMtuutbGlV8ivfK1I47Jxi2hQ%3D&st=2020-10-22T20%3A10%3A20Z&se=2020-11-22T20%3A15%3A20Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/excel/custom-functions-json.md [hunk 1]
```diff
@@ -150,12 +151,12 @@ The `functions` property is an array of custom function objects. The following t
...
+| Property          | Data type | Required                               | Description |
 | :---------------- | :-------- | :------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/custom-functions-json.md:155]: At least 3 dashes are needed but you don't need the number of dashes to match the maximum number of characters in a subsequent row for that column.
```suggestion
| :---------------- | :-------- | :------------------------------------- | :---------- |
```

File: docs/excel/custom-functions-json.md [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
 
 [!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
 
-We recommend using JSON auto-generation when possible instead of creating your own JSON file. Auto-generation is less prone to user error and the `yo office` scaffolded files already include this. For more information on the process of JSDoc comment JSON file generation, see [Generate JSON metadata for custom functions](custom-functions-json-autogeneration.md).
+We recommend using JSON autogeneration when possible instead of creating your own JSON file. Autogeneration is less prone to user error and the `yo office` scaffolded files already include this. For more information on JSDoc tags and the JSON autogeneration process, see [Autogenerate JSON metadata for custom functions](custom-functions-json-autogeneration.md).
 
 However, you can make a custom functions project from scratch; it requires that you:
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/custom-functions-json.md:16]:
```suggestion
However, you can make a custom functions project from scratch but it requires you to:
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Just a couple of suggestions.

File: docs/excel/custom-functions-json.md (hunk 1 above)
Comment (alison-mk) [reply docs/excel/custom-functions-json.md:155]: Sweet! Thanks, I wondered why there were so many dashes here.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c82d0b9](https://github.com/OfficeDev/office-js-docs-pr/commits/c82d0b92690f33d3c911323de7e15c0e4dc302d0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-custom-functions-json-cleanup/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2273) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202010222236025516-2273%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=8JsgIN0SaHYXHh5HIqyrzxEGvzByXFs1UjS0J247304%3D&st=2020-10-22T22%3A31%3A42Z&se=2020-11-22T22%3A36%3A42Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by mscharlock — APPROVED** [review]

**Pull request merged**